### ISSUE TITLE
Review -> Can't add new method as reviewer

### ISSUE THREAD

**Issue by clegarde** [spawaskar-cora/cora-docs#508]
If the person inventorying did not add a method, the reviewer cannot add a new method. They should be able to add a method in case the initial person forgot/neglected it.

![Screen Shot 2019-08-14 at 9 06 58 PM](https://user-images.githubusercontent.com/7350474/63068184-d3131500-bed7-11e9-927a-e309711a80c6.png)

**Comment by fedamann** [comment]
What precludes the reviewer from talking to the analyst and recommending
the method be added ?  Method selection is preview of analyst not the
reviewer.

**Comment by clegarde** [comment]
For example - Ploesti project is adding McKern and Stewart epiphyseal fusion scoring, so if the analyst forgets, they need to add it. It's a project directed biz rule, not like an FAR that is up to the analyst. If we have a review system like this, it would be nice to allow for all parts of the process to be in it, rather than have certain aspects in the system and other parts have to be noted and discussed. This isn't necessarily taking away communicating with each other, but it is a system to work through and accept or not in the system. If you're inventorying 100 specimens, it just adds more work for the reviewer if they need to make additional notes outside the system.

**Comment by fedamann** [comment]
If it's a business rule for one or two projects then the team members need
to adhere to it. If they don't then they need to go back in and fix it.
If/when this happens the analyst will not forget the next time. We don't
need code changes for this to be managed.  This is a low priority change.

**Comment by clegarde** [comment]
Right. They *should*. And the purpose of a review process is to indicate that they haven’t done something, need to change something, or they did it right. There should be a way to indicate a change is necessary without having to write notes outside the system. 
A suggested future enhancement for the review process is to have an option as the reviewer to “Agree” or “Disagree” for each component. This would ensure the reviewer has checked everything they need to as well as allow for the reviewer to indicate that there’s an issue without adding new information. This enhancement was discussed with a small group and noted for future discussions with the group for enhancing and streamlining the review component. Also briefly mentioned to Sachin.

**Comment by fedamann** [comment]
Carrie, thank you for your feedback.

**Comment by clegarde** [comment]
Same as #421

**Comment by lfkennedy** [comment]
For M47, this would be nice to have. Business rules for the project is to mark fusion state. As the peer reviewer, we can send emails, but if an analyst forgot to mark fusion of one element, and we are attempting to make the peer review completely in-house within CoRA, the peer reviewer should be able to add suggested methods rather than having to go outside the program.

**Comment by lfkennedy** [comment]
@SachinPawaskarUNO has said that a comment section may be possible to add to the reviewer space within CoRA. This would be helpful for reviewers to have in order to add/suggest new methods.
